l10n_nl_partner_name/data/ir.config_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+    <record id="param_name_order" model="ir.config_parameter">
```

Comment:
Where is this used?

Reply:
Basically nowhere, in the 10.0 version of the module either. I guess it is used to force name order for res.partner to this specific format, when the module is installed

Reply:
I found it. It is defined in partner_firstname. Problem is that this module does not respect the setting defined there.